build/css/src-base/_colors.css

```diff
@@ -2,26 +2,246 @@
 # Color Classes
 --------------------------------------------------------------*/
 
-.has-primary-color {
-	color: var(--primary-color);
+/** Color: primary-300 */
+:root .has-primary-300-color {
+	color: var(--color-primary-300);
```

Comment:
Instead of `attr: var(hex)` a different approach could be to store `r, g, b` to the color vars and use them like `attr: rgb|a(var(--color-key-hue?)|, 0.69)`.
This makes the use of colors more flexible

Reply:
The added complexity is probably not worth it for our standard use case.